### DISCUSSION TITLE
Space ROS Technical Meeting Agenda 2024-03-21

### DISCUSSION THREAD

**Discussion by Bckempa** [space-ros/space-ros#144] [General]
Previous agenda: #141

### Agenda
- [x] Introduce any new attendees.
- [x] Summarize any PRs merged since last meeting.
- [ ] Review status of [next release](https://github.com/orgs/space-ros/projects/3/views/8)
- [ ] Open next meeting discussion

**Comment by ivanperez-keera** [comment]
Opened next meeting agenda at #150 .
